gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -120,15 +123,15 @@ def get_network_functions(self, context, filters=None):
 
     @log_helpers.log_method_call
     def update_network_function(self, context, network_function_id,
-                                updated_network_function):
+                                config):
```

Comment:
This method is invoked in the node driver with config=context.current_node instead of config=context.current_node['config']

Is it expected, then in that case the argument name can be network_function (modified one, like context.current_node)